packages/recomponents/src/components/r-repeater/r-repeater.vue

```diff
@@ -77,12 +78,14 @@
                 }
             },
         },
-        created() {
+        mounted() {
             this.fetchData();
         },
         watch: {
-            watcher() {
-                this.fetchData();
+            watcher(newValue, oldValue) {
+                if (isEqual(newValue, oldValue)) {
+                    this.fetchData();
+                }
```

Comment:
is it makes sense to add deep watch here?

Reply:
seems like does

Reply:
Yes :) thank you!